build.psm1

```diff
@@ -386,7 +386,7 @@ Fix steps:
             # Relative paths do not work well if cwd is not changed to project
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-linux`
Clipboard cmdlet tests.Text.Get-Clipboard returns what is in Set-Clipboard
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 18
18:             Set-Clipboard -Value $guid

```
Clipboard cmdlet tests.Text.Get-Clipboard returns an array
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 25
25:             1,2 | Set-Clipboard

```
Clipboard cmdlet tests.Text.Get-Clipboard -Raw returns one item
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 33
33:             1,2 | Set-Clipboard

```
Clipboard cmdlet tests.Text.Set-Clipboard -Append will add text
```powershell
Operation is not supported on this platform.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 39
39:             'hello' | Set-Clipboard

```